lms/djangoapps/discussion_api/api.py

```diff
@@ -138,6 +161,26 @@ def get_thread_list(request, course_key, page, page_size):
     # behavior and return a 404 in that case
     if result_page != page:
         raise Http404
+    staff_user_ids = {
+        user.id
+        for role in Role.objects.filter(
+            name__in=[FORUM_ROLE_ADMINISTRATOR, FORUM_ROLE_MODERATOR],
+            course_id=course_key
+        )
+        for user in role.users.all()
+    }
+    ta_user_ids = {
+        user.id
+        for role in Role.objects.filter(name=FORUM_ROLE_COMMUNITY_TA, course_id=course_key)
+        for user in role.users.all()
+    }
+    group_ids_to_names = {
+        group.id: group.name
+        for group in CourseUserGroup.objects.filter(course_id=course_key)
```

Comment:
instead of importing and using the CourseUserGroup model directly here, please add a new api function to course_groups.cohorts that returns this dictionary.